_posts/en/newsletters/2023-11-22-newsletter.md

```diff
@@ -0,0 +1,183 @@
+      includes an offer for Alice.  Another record,
+      `bob._lnaddress.domain.com`includes an offer for Bob.  Teinturier
+      notes that this requires the domain owner create one DNS record
+      per user (and update that record if the user needs to change their
+      default offer).
+
+  As of this writing, there had not been any public discussion of the
+  ideas.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **BitMask Wallet 0.6.3 released:**
+  [BitMask][bitmask website] is a web and browser extension-based wallet for Bitcoin,
+  Lightning, RGB, and [payjoin][topic payjoin].
+
+- **Opcode documentation website announced:**
+  The [https://opcodeexplained.com/] website was recently [announced][OE tweet]
+  and provides explanations of many of Bitcoin's opcodes. The effort is ongoing
+  and [contributions are welcome][OE github].
+
+- **Athena Bitcoin adds Lightning support:**
+  The Bitcoin ATM [operator][athena website] recently [announced][athena tweet]
+  support for receiving Lightning payments for cash withdrawals.
+
+- **Blixt v0.6.9 released:**
+  The [v0.6.9][blixt v0.6.9] release includes support for simple taproot
+  channels, defaults to [bech32m][topic bech32] receive addresses, and adds
+  additional [zero conf channel][topic zero-conf channels] support.
+
+- **Durabit whitepaper announced:**
+  The [Durabit whitepaper][] outlines a protocol using [timelocked][topic
+  timelocks] Bitcoin transactions in conjunction with a chaumian-style mint
+  to incentivize seeding of large files.
+
+- **BitStream whitepaper announced:**
+  The [BitStream whitepaper][] and [early prototype][bitstream github] layout a
+  protocol for the hosting and atomic exchange of digital content for
+  coins using timelocks and merkle trees with verification and fraud
+  proofs.  For previous discusison of paid data transfer protocols, see
+  [Newsletter #53][news53 data].
+
+- **BitVM proof of concepts:**
+  Two proof of concepts building on [BitVM][news273 bitvm] were posted including
+  one [implementing][bitvm tweet blake3] the [BLAKE3][] hash function and
+  [another][bitvm techmix poc] that [implements][bitvm sha256] SHA256.
+
+- **Bitkit adds taproot send support:**
+  [Bitkit][bitkit website], a mobile Bitcoin and Lightning wallet, added
+  [taproot][topic taproot] sending support in the [v1.0.0-beta.86][bitkit
+  v1.0.0-beta.86] release.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.17.2-beta][] is a maintenance release that only includes one
+  small change to fix the bug reported in [LND #8186][].
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full node implementation. There's a [testing
```

Comment:
```suggestion
  version of the predominant full-node implementation. There's a [testing
```

Reply:
This doesn't look right. A grep through the `_posts/en` directory gives 240 results for "full node" and only 6 for "full-node", all of which are a part of a URL.

Reply:
I don't have a strong preference either way.  I suggest we leave it as-is in the merged newsletter and I'll try to remember this discussion the next time someone suggests the hyphenated / single-adjective version and we can debate it then.